src/sidebar/components/annotation.js

```diff
@@ -444,6 +444,9 @@ function AnnotationController(
   };
 
   this.tagSearchURL = function(tag) {
+    if (this.isThirdPartyUser()) {
+      return null;
+    }
```

Comment:
I didn't have to change anything in the template. It uses ngHref: `<a ng-href="{{vm.tagSearchURL(tag)}}" target="_blank">{{tag}}</a>` so that if `tagSearchURL()` returns null it renders an `<a>` with an `ng-href` but with no `href` attribute.

Reply:
I wondered about that. Thanks for noting this.